rpm/macros.kernel-source

```diff
@@ -49,6 +49,45 @@ esac)
 
 %linux_make_arch ARCH=%linux_arch
 
+# no args -> /usr/src/linux
```

Comment:
What would this be used for?

This symlink points to some random place. IT should not be used for anything that's intended to be reproducible.

Reply:
It should point to some kernel source unless something goes wrong.

I don't know where you get the guarantee that it points to the latest. I am not aware of any. Maybe more like last installed.